howto/index.md

```diff
@@ -23,9 +23,8 @@ taking you through the setup of your own Charmed HPC cluster.
 (howto-manage)=
 ## Manage
 
-The how-to guides in this section show you how to perform common management tasks after you have
-deployed Charmed HPC. These guides are for system administrators and HPC engineers that are
-responsible for managing Charmed HPC after deployment.
+The how-to guides in this section show you how to integrate with various optional services and perform common management tasks after you have
```

Comment:
I don't think we need various here. We can just scope to "optional services" so people don't think there's a whole universe of applications they can integrate with :sweat_smile: 

```suggestion
The how-to guides in this section show you how to integrate with optional services and perform common management tasks after you have
```